recipes/namd/3.0/gh200/compilers.yaml

```diff
@@ -0,0 +1,5 @@
+bootstrap:
+  spec: gcc@12
+gcc:
+  specs:
+  - gcc@12
```

Comment:
The version of gcc won't change, e.g. from `12.3` to `12.4` so long as the version of spack used to build the image does not change.
But it can be useful for porting to a new version to record more precisely which version of gcc was being used.